### PR TITLE
feat: add ack example cluster

### PR DESCRIPTION
**Pull request by jwcesign**
Fixes #none

## Proposed Changes

* feat: add ack example cluster

**Release Note**

```release-note

```

### CODE REVIEW

**Comment by jwcesign** [comment]
cc @helen-frank

File: clusters/ack/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+cd examples/clusters/ack
+```
+Execute the following command to create the ACK cluster:
+```bash
+terraform init
+terraform apply --auto-approve
+```
+
+After finish the process, run the following command to get the `kubeconfig`:
+```sh
```
Comment (helen-frank) [inline clusters/ack/README.md]: use `bash`

File: clusters/ack/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+export KUBECONFIG=~/.kube/demo
+export CLUSTER_ID=$(aliyun cs GET /clusters | jq -r '.[] | select(.name == "cluster-demostration") | .cluster_id')
+aliyun cs GET /k8s/$CLUSTER_ID/user_config | jq -r '.config' > $KUBECONFIG
+```
+
+Then testing the cluster:
+```sh
```
Comment (helen-frank) [inline clusters/ack/README.md]: use `bash`

File: clusters/ack/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+
+Using this tutorial, you can create a demonstration Aliyun cluster to try CloudPilot AI.
+
+### Prerequisites
+
+- [aliyun cli](https://help.aliyun.com/zh/cli/?spm=a2c4g.11186623.0.0.18ec478dVSmS6M)
+  
+  If you use Linux, run the following command to install the aliyun cli:
+  ```sh
```
Comment (helen-frank) [inline clusters/ack/README.md]: use `bash`

**Review by helen-frank — COMMENTED** [review]
`sh` or `bash`

Other /lgtm

**Pull request merged**